trieste/acquisition/function/active_learning.py

```diff
@@ -245,3 +246,78 @@ def acquisition(x: TensorType) -> TensorType:
         return criterion
 
     return acquisition
+
+class BayesianActiveLearningByDisagreement(SingleModelAcquisitionBuilder):
+    """
+    Builder for the *Bayesian Active Learning By Disagreement* acquisition function defined in
```

Comment:
acq fnc is the class below, this is a builder so leave the first sentence here, but move the rest to the docstring in the class below

Reply:
moved